src/PersistentSubscriptions/PersistentSubscriptionsManager.php

```diff
@@ -36,9 +36,10 @@ public function __construct(
         EndPoint $httpEndPoint,
         int $operationTimeout,
         string $httpSchema = EndpointExtensions::HTTP_SCHEMA,
-        ?UserCredentials $defaultUserCredentials = null
+        ?UserCredentials $defaultUserCredentials = null,
+        bool $verifyPeer = true
```

Comment:
please keep `$defaultUserCredentials` the last argument, move `$verifyPeer` further up instead.